uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/components/my-leave-calendar.html

```diff
@@ -0,0 +1,386 @@
+<section>
```

Comment:
I'd rather leave this static, as the ticket implies. When the controller will be properly developed, the template will use all the angular directives it needs